.gitignore

```diff
@@ -5,4 +5,6 @@ bin/
 docs/html/*
 obj/
 include/
```

Comment:
This include folder was only used for the nlohmann json. Since that's now under submodules, I think we can probably get rid of it altogether

Reply:
Thanks: dropped it.